README.md

```diff
@@ -1 +1,63 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> Description the project.
+
+
+## Built With
+
+- Html And Css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
```

Comment:
- [ ] Kindly write the steps the user should follow to install your repo like writing `git clone https://github.com/Blossom-Aniedi/hello-microverse.git` in the terminal.